src/uwtools/tests/test_schemas.py

```diff
@@ -1889,16 +1892,19 @@ def test_schema_shave():
 def test_schema_shave_config_properties():
     # Get errors function from schema_validator
     errors = schema_validator("shave", "properties", "shave", "properties", "config")
-    for key in ("input_grid_file", "nx", "ny", "nh4"):
+    for key in ("input_grid_file", "nx", "ny", "nhalo"):
         # All config keys are required:
         assert f"'{key}' is a required property" in errors({})
         # A string value is ok for input_grid_file:
         if key == "input_grid_file":
             assert "not of type 'string'" in str(errors({key: 42}))
-        # nx, ny, and nh4 must be positive integers:
-        elif key in ["nx", "ny", "nh4"]:
+        # nx, ny, and nhalo must be integers:
+        elif key in (keyvals := {"nx": 1, "ny": 1, "nhalo": 0}):
+            minval = keyvals[key]
             assert "not of type 'integer'" in str(errors({key: "/path/"}))
-            assert "0 is less than the minimum of 1" in str(errors({key: 0}))
+            assert f"{minval - 1} is less than the minimum of {minval}" in str(
+                errors({key: minval - 1})
+            )
```

Comment:
Looking for feedback that this new approach is okay after setting nhalo's min val to 0. It somehow felt like I was being bad, but maybe not?

Reply:
Looks like a nice test to me.